packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -490,6 +491,10 @@ class TextStyle extends Diagnosticable {
   /// The style in which to paint the text decorations (e.g., dashed).
   final TextDecorationStyle decorationStyle;
 
+  /// The thickness of the decoration as a muliplier of the thickness specified
+  /// by the font.
```

Comment:
this needs to be more detailed. What does "thickness" mean? What is the default? Maybe a screenshot would explain it better? (see the assets-for-api-docs repo for how to make diagrams)